nemo/collections/llm/api.py

```diff
@@ -311,22 +311,22 @@ def deploy(
         output_generation_logits (bool): If True builds trtllm engine with gather_generation_logits set to True.
         generation_logits are used to compute the logProb of the output token. Default: True.
     """
-    from nemo.collections.llm import deploy
+    from nemo.collections.llm.deploy.base import get_trtllm_deployable, unset_environment_variables
```

Comment:
is this the main fix, to be specific about imports? this seems to have come up in a few other issues, so should nemo move entirely to be as specific as possible for such imports?

Reply:
Yes this is the main fix. I updated the PR description to reflect this.